src/common/utils/setup-helpers.js

```diff
@@ -135,8 +141,11 @@ function setStorageConfig(options: KPOptionsObject): void {
  * @returns {void}
  */
 function applyStorageSupport(player: KalturaPlayer): void {
-  if (StorageManager.isLocalStorageAvailable()) {
-    StorageManager.attach(player);
+  if (LocalStorageManager.isStorageAvailable()) {
```

Comment:
why does at this level, we need to know about storage types? why can we call - baseStorageManager::attachAll()
And it will check all available children and will attach each?